lib/pharos/host/configurer.rb

```diff
@@ -1,15 +1,21 @@
 # frozen_string_literal: true
 
+require 'set'
+
 module Pharos
   module Host
     class Configurer
-      attr_reader :host, :ssh
+      attr_reader :host, :config
 
       SCRIPT_LIBRARY = File.join(__dir__, '..', 'scripts', 'pharos.sh').freeze
 
-      def initialize(host, ssh)
+      def initialize(host, config = nil)
```

Comment:
What is the `config` here?

Reply:
Instead of the `configurers.each { |c| c.cluster_config = config }` the configurer gets the config via initialize.

Reply:
Ok, we should add yardoc for more pleasant reading experience.